source/extensions/filters/http/dynamo/dynamo_utility.h

```diff
@@ -9,6 +9,15 @@ namespace Dynamo {
 
 class Utility {
 public:
+  // returns example max name length, 127. This number is easily mutable in the
+  // outside world, and depending on it makes this test fragile. Thus, the examples
```

Comment:
This doesn't look like a test utility; this looks like production code.

Reply:
You're right. I messed up here -- thinking it was a utility which existed solely for `dynamo_utility_test.cc`. I think the thing breaking this test is a leaky stat